packages/platform-ios/native_modules.rb

```diff
@@ -15,7 +15,8 @@ def use_native_modules!(config = nil)
     config = nil;
   end
 
-  cli_resolve_script = "console.log(require('@react-native-community/cli').bin);"
+  # Resolving the path the RN CLI. The `@react-native-community/cli` module may not be there for certain package managers, so we fall back to resolving it through `react-native` package, that's always present in RN projects
+  cli_resolve_script = "try {console.log(require('@react-native-community/cli').bin);} catch (e) {console.log(require('react-native/cli').bin);}"
```

Comment:
cc @alloy in reality this try/catch is only necessary for testing environment. Do you think it's better to leave this or somehow detect we're in "test" environment and change path accordingly? (usually a bad idea in my experience)

Reply:
Might be better to mock `execute_command` instead?